pkgs/development/python-modules/nettigo-air-monitor/default.nix

```
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "nettigo-air-monitor";
-  version = "2.2.0";
+  version = "2.2.1";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```

And add `setuptools` to `nativeBuildInputs`. Running `python setup.py install` is deprecated. Every update should migrate packages away from setuptools CLI usage.